src/rendering/defaults.rs

```diff
@@ -13,18 +13,18 @@ pub fn get_random_walk(r: f32,g: f32,b: f32,n: i32) -> Line{
     let mut y = 0.0;
 
     let mut rng = rand::thread_rng();
-    
+
     for x in 0..n {
-        let v = Vertex{
-            position: [(x as f32)/1000.0, y, 0.0, 1.0],
-            color: [r,g,b,0.0],
+        let v = Vertex {
+            position: [(x as f32) / 1000.0, y, 0.0, 1.0],
```

Comment:
Magic numbers should be variables with specific names, or you should have a comment saying why these numbers are what they are.

Just from the function name, my guess is this is for random points and data?  But still, keep up the same practice through the code base (yes not super important, but for new contributors that jump in it'll show them what you're expecting)